### PR TITLE
bugfix installed package list in its detail's page when the package's name has spaces

### PR DESCRIPTION
**Pull request by goldsky**
Rebase https://github.com/modxcms/revolution/pull/12551 to 2.x

### CODE REVIEW

File: manager/assets/modext/workspace/package.grid.js [hunk 1]
```diff
@@ -237,10 +237,10 @@ Ext.extend(MODx.grid.Package,MODx.grid.Grid,{
 	/* Install a package */
 	,install: function( record ){
 		Ext.Ajax.request({
-			url : MODx.config.connectors_url
+			url : MODx.config.connector_url
 			,params : {
 				action : 'workspace/packages/getAttribute'
-				,attributes: 'license,readme,changelog,setup-options,requires'
+				,attributes: 'license,readme,changelog,setup-options'
```
Comment (theboxer) [inline manager/assets/modext/workspace/package.grid.js]: Why you are removing package dependencies?

File: manager/assets/modext/workspace/package.grid.js [hunk 2]
```diff
@@ -161,7 +161,7 @@ Ext.extend(MODx.grid.Package,MODx.grid.Grid,{
     	};
         Ext.getCmp('modx-package-search').reset();
     	this.getBottomToolbar().changePage(1);
-        //this.refresh();
+        this.refresh();
```
Comment (theboxer) [inline manager/assets/modext/workspace/package.grid.js]: Unnecessary refresh(), produces canceled request

File: manager/assets/modext/workspace/package.grid.js [hunk 3]
```diff
@@ -151,7 +151,7 @@ Ext.extend(MODx.grid.Package,MODx.grid.Grid,{
         var nv = newValue || tf;
         this.getStore().baseParams.search = Ext.isEmpty(nv) || Ext.isObject(nv) ? '' : nv;
         this.getBottomToolbar().changePage(1);
-        //this.refresh();
+        this.refresh();
```
Comment (theboxer) [inline manager/assets/modext/workspace/package.grid.js]: Unnecessary refresh(), produces canceled request

File: manager/assets/modext/workspace/package.grid.js (hunk 1 above)
Comment (goldsky) [inline manager/assets/modext/workspace/package.grid.js]: ow, that's new, I'll revert it back

**Comment by goldsky** [comment]
updated

**Comment by theboxer** [comment]
I don't think this PR fixes the problem you described, at least for me. I still don't see the previous versions in detail panel.

**Comment by goldsky** [comment]
try now

**Pull request merged**